youtube_base/youtubers/views.py

```diff
@@ -148,24 +149,44 @@ def post(self, request, *args, **kwargs):
 
         """
         form = CategoryForm(request.POST)
+        tag_form = TagForm(request.POST)
+
         if form.is_valid():
             categories = form.cleaned_data['categories']
             youtubers = Youtuber.objects.filter(categories__in=categories)
-
```

Comment:
I'd recommend not to do changes + refactorings within a single commit.

1. Refactoring commit.
2. Additional changes

Or visa versa.